exonum-java-binding/service-archetype/src/main/resources-filtered/archetype-resources/__rootArtifactId__-messages/pom.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>\${groupId}</groupId>
+    <artifactId>\${rootArtifactId}-parent</artifactId>
+    <version>\${version}</version>
+  </parent>
+
+  <artifactId>\${rootArtifactId}-messages</artifactId>
+  <version>\${version}</version>
+  <packaging>jar</packaging>
+
+  <name>Exonum Service Messages</name>
+
+  <properties>
+    <!--
+    Use clientRelease for messages, as arbitrary clients may need the messages.
+    -->
+    <java.compiler.release>\${java.compiler.clientRelease}</java.compiler.release>
+    <os-maven-plugin.version>${afp.os-maven-plugin.version}</os-maven-plugin.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java</artifactId>
+    </dependency>
+
+    <!--
+    TODO: Shall we include them, if by default Exonum types are not really needed
+      (e.g., proto-definitions in QA & Crypto do not use them)?
```

Comment:
⚠️ 
---
On the other hand, if you need proofs, you need these definitions.

Reply:
Not sure that relying on Exonum messages is really needed in service messages.
Also, this dependency wouldn't work for other languages (because you need raw protos).

Reply:
If you transfer proofs in protobuf, you do need Exonum messages:

https://github.com/exonum/exonum-java-binding/blob/2ee756cbd09a70db591d7362fdeeffde6476f29e/exonum-java-binding/core/src/main/java/com/exonum/binding/core/blockchain/Blockchain.java#L102-L109